src/main/resources/application.yaml

```diff
@@ -332,13 +332,22 @@ idam.s2s-auth.microservice: ${IA_S2S_MICROSERVICE:ia}
 idam.s2s-auth.url: ${S2S_URL:http://127.0.0.1:4502}
 
 feign:
+  hystrix:
+    enabled: true
```

Comment:
I am pretty sure that this will not break our functionalities but could you change this setting only for Case Role feign clients?
we have a few in case-api, not sure if we want to use circuit breaker by default in all of them:
https://cloud.spring.io/spring-cloud-openfeign/reference/html/#spring-cloud-feign-hystrix